genesyscloud/resource_genesyscloud_flow.go

```diff
@@ -74,32 +73,31 @@ func ResourceFlow() *schema.Resource {
 				Type:         schema.TypeString,
 				Required:     true,
 				ValidateFunc: ValidatePath,
+				ForceNew:     true,
 			},
 			"file_content_hash": {
 				Description: "Hash value of the YAML file content. Used to detect changes.",
 				Type:        schema.TypeString,
 				Required:    true,
+				ForceNew:    true,
```

Comment:
So trying to understand this.  We added force new to everything.  By adding a forceNew does this cause the file_content_hash to always be reset even if the deploy fails?

Reply:
Yes. Because it's all ForceNew, when the user tries to update something terraform will tear down the existing flow, wiping the state file, then fail a create. If the user re-runs `terraform apply` it will try again at a create.